tests/settlement_test.go

```diff
@@ -1,22 +1,90 @@
 package tests
 
 import (
-    "context"
-    "github.com/craftgate/craftgate-go-client/adapter"
-    craftgate "github.com/craftgate/craftgate-go-client/adapter"
-    "github.com/davecgh/go-spew/spew"
-    "testing"
+	"context"
+	"github.com/craftgate/craftgate-go-client/adapter"
+	craftgate "github.com/craftgate/craftgate-go-client/adapter"
```

Comment:
import order can be change to standart

Reply:
```
import (
	"context"
	"testing"
	
	"github.com/davecgh/go-spew/spew"
	
	craftgate "github.com/craftgate/craftgate-go-client/adapter"
)
```

I said that this format is the most used format. You can also use the -local flag if you want. No big deal just to get close to the standard